### PR TITLE
Add vddk support for more esx cases

### PR DESCRIPTION
**Pull request by xiaodwan**
1) Add vddk support for convert_vm_to_libvirt
2) Add vddk support for function_test_esx

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_5.linux.latest7.arch_x86_64.it_vddk PASS (378.17 s)
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.uefi.ovmf.rhel.esx_55.libvirt.it_vddk: PASS (649.34 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh please help to review it, thanks.

File: v2v/tests/cfg/convert_vm_to_libvirt.cfg [hunk 1]
```diff
@@ -16,6 +16,16 @@
...
+    variants:
+        - it_vddk:
+            only source_esx
+            input_transport = 'vddk'
+            vddk_libdir_src = MOUNT_SRC_VDDK_LIB_DIR_V2V_EXAMPLE
+            # Set the real vddk_libdir/thumbprint or keep it commented
+            #vddk_libdir = VDDK_LIB_DIR_EXAMPLE
+            #vddk_thumbprint = VDDK_THUMBPRINT_EXAMPLE
+        - it_default:
+
```
Comment (chunfuwen) [inline v2v/tests/cfg/convert_vm_to_libvirt.cfg]: remove additional line

Comment (xiaodwan) [reply v2v/tests/cfg/convert_vm_to_libvirt.cfg]: Done

**Pull request merged**